Update UI

AnBui2004 opened the pull request:
<img width="1466" height="3101" alt="Screenshot_20250725_205823" src="https://github.com/user-attachments/assets/6b877072-d81f-4c51-8657-3a616f90023a" />
<img width="1466" height="3101" alt="Screenshot_20250725_211241" src="https://github.com/user-attachments/assets/3ed28ae4-c43e-42ad-aa76-03e9886f2f57" />
<img width="1466" height="3101" alt="Screenshot_20250725_211402" src="https://github.com/user-attachments/assets/10555368-bba5-45c0-878b-9a230e6f5ad3" />
<img width="1466" height="3101" alt="Screenshot_20250725_211715" src="https://github.com/user-attachments/assets/fd0d2a94-4387-4dbe-8f14-14d2fc66eb4c" />
<img width="1466" height="3101" alt="Screenshot_20250725_211739" src="https://github.com/user-attachments/assets/7706066c-4d5e-4e9b-8782-07107fc2a8eb" />
<img width="899" height="420" alt="image" src="https://github.com/user-attachments/assets/a40193c8-3228-4343-bdb2-a5369e26c8e1" />
<img width="780" height="530" alt="image" src="https://github.com/user-attachments/assets/0222869a-6f6c-4525-aaaf-fc469d648c00" />

AnBui2004 commented:
[Screen_recording_20250725_212326.webm](https://github.com/user-attachments/assets/a1c42abf-02d9-4506-a9be-1b9b18d4874f)

StudioAsInc commented:
I see. You added new package classes. I request you to mention all your changes in the description cause they look exciting to me 💣

AnBui2004 commented:
- LoadingIndicator for Projects and Store in MainActivity.java.
- Show error when data loading error in Store occurs.
- Change icons in ExportProject.java and export_project.xml.
- Change Discord, Telegram, GitHub icons in drawer in MainActivity and about.xml

AnBui2004 commented:
`feat: Added CollapsingToolbarLayout in activity_app_settings.xml`
[Screen_recording_20250725_233956.webm](https://github.com/user-attachments/assets/7828dedb-f933-480d-a2cc-d6f9d9353b93)

yoann256 commented:
> `feat: Added CollapsingToolbarLayout in activity_app_settings.xml` [Screen_recording_20250725_233956.webm](https://github.com/user-attachments/assets/7828dedb-f933-480d-a2cc-d6f9d9353b93)

you have to put these in the commit msg

AnBui2004 commented:
Changes:
`feat: Added LoadingIndicator for Projects and Store in MainActivity.java.`
`feat: Show error message when data loading in Store fails.`
`style: Changed icons in ExportProject.java and export_project.xml.`
`style: Changed Discord, Telegram, GitHub icons in drawer in MainActivity and about.xml.`
`feat: Added CollapsingToolbarLayout in activity_app_settings.xml.`

Just changed:
`feat: Added Try again button for Store.`

AnBui2004 commented:
https://github.com/user-attachments/assets/48117f1a-a957-4b66-9c8a-22429236a184

Changes:
`feat: Added LoadingIndicator for Projects and Store in MainActivity.java.`
`feat: Show error message when data loading in Store fails.`
`style: Changed icons in ExportProject.java and export_project.xml.`
`style: Changed Discord, Telegram, GitHub icons in drawer in MainActivity and about.xml.`
`feat: Added CollapsingToolbarLayout in activity_app_settings.xml.`
`feat: Added Try again button for Store.`

Just changed:
`style: Changed CircularProgressIndicator to LoadingIndicator in fragment_beta_changes.xml. and fragment_about_app.xml.`

MadaraMods commented:
> If you don't follow contribution guidelines, your PRs will not be merged.

Bro what problem in here the code is clean and the UI is pretty nice

AnBui2004 commented:
https://github.com/user-attachments/assets/5c30c3b9-9636-47ca-b502-6efe5d40ce38

`style: Updated interface for Code Shrinking Manager.`

AnBui2004 commented:
Changes:
`feat: Added LoadingIndicator for Projects and Store in MainActivity.java.`
`feat: Show error message when data loading in Store fails.`
`style: Changed icons in ExportProject.java and export_project.xml.`
`style: Changed Discord, Telegram, GitHub icons in drawer in MainActivity and about.xml.`
`feat: Added CollapsingToolbarLayout in activity_app_settings.xml.`
`feat: Added Try again button for Store.`
`style: Changed CircularProgressIndicator to LoadingIndicator in fragment_beta_changes.xml. and fragment_about_app.xml.`

Just changed:
`style: Updated interface for Code Shrinking Manager.`

AnBui2004 commented:
<img width="1466" height="3101" alt="image" src="https://github.com/user-attachments/assets/9c4cb216-9647-4a7b-842d-3259849b995a" />
<img width="1466" height="3101" alt="image" src="https://github.com/user-attachments/assets/e47e0e86-9c24-4279-a591-3ff33b1aeaaf" />

AnBui2004 commented:
Changes:
`feat: Added LoadingIndicator for Projects and Store in MainActivity.java.`
`feat: Show error message when data loading in Store fails.`
`style: Changed icons in ExportProject.java and export_project.xml.`
`style: Changed Discord, Telegram, GitHub icons in drawer in MainActivity and about.xml.`
`feat: Added CollapsingToolbarLayout in activity_app_settings.xml.`
`feat: Added Try again button for Store.`
`style: Changed CircularProgressIndicator to LoadingIndicator in fragment_beta_changes.xml. and fragment_about_app.xml.`

Just changed:
`style: Updated interface for Code Shrinking Manager.`

StudioAsInc commented:
### Please redesign it...

![Screenshot_20250726_202006_GitHub.jpg](https://github.com/user-attachments/assets/bcaf803d-b414-4176-8170-0b9cf0326bae)

AnBui2004 commented:
> You're making mistakes even after warning by The team
> 
> ### **Why Your Commits Violate Sketchware Pro’s Guidelines**
> 1. **Missing Prefixes**
>    
>    * Commits must start with `feat:`, `fix:`, `style:`, etc.
>    * Your messages (e.g., _"Updated interface for Code Shrinking..."_) lack these.
> 2. **Vague & Repetitive Messages**
>    
>    * The same commit message appears multiple times in short intervals (e.g., 36m, 47m, 53m ago).
>    * No details on _what_ was updated—just "Updated interface."
> 3. **Too Many Small Commits**
>    
>    * Rapid, minor changes should be **squashed** into a single descriptive commit.(Specifically if you use IDE)
> 4. **Incorrect Merge Message**
>    
>    * _"Merge branch 'Sketchware-Pro:main'..."_ doesn’t follow conventions.
> 
> ### **How to fix it**
> * **Prefix properly** (e.g., `feat: Improve Code Shrinking UI`).
> * **Group related changes** into one commit.
> * **Be specific**—explain _what_ and _why_ you changed something.
> 
> Follow the **[Contribution Guidelines](https://github.com/Sketchware-Pro/Sketchware-Pro/blob/main/CONTRIBUTING.md)** to avoid PR rejections.

I'm right because that's all the changes in those times, should I just change the interface here and have to add the changes that didn't change at all? And what is this?
<img width="1600" height="900" alt="image" src="https://github.com/user-attachments/assets/6ea7c331-e5b5-4e9b-a045-2e6d8c0225f1" />

AnBui2004 commented:
> You're making mistakes even after warning by The team
> 
> ### **Why Your Commits Violate Sketchware Pro’s Guidelines**
> 1. **Missing Prefixes**
>    
>    * Commits must start with `feat:`, `fix:`, `style:`, etc.
>    * Your messages (e.g., _"Updated interface for Code Shrinking..."_) lack these.
> 2. **Vague & Repetitive Messages**
>    
>    * The same commit message appears multiple times in short intervals (e.g., 36m, 47m, 53m ago).
>    * No details on _what_ was updated—just "Updated interface."
> 3. **Too Many Small Commits**
>    
>    * Rapid, minor changes should be **squashed** into a single descriptive commit.(Specifically if you use IDE)
> 4. **Incorrect Merge Message**
>    
>    * _"Merge branch 'Sketchware-Pro:main'..."_ doesn’t follow conventions.
> 
> ### **How to fix it**
> * **Prefix properly** (e.g., `feat: Improve Code Shrinking UI`).
> * **Group related changes** into one commit.
> * **Be specific**—explain _what_ and _why_ you changed something.
> 
> Follow the **[Contribution Guidelines](https://github.com/Sketchware-Pro/Sketchware-Pro/blob/main/CONTRIBUTING.md)** to avoid PR rejections.

This rule isn’t really useless because GitHub makes it very easy to view changes to code. That’s why few people bother with this rule.

AnBui2004 commented:
I've compiled all the changes here every time there's a change, right? The latest changes are all improvements over the previous ones. Shouldn't you at least be able to read the code and see what's changed?

StudioAsInc commented:
> This rule isn’t really useless because GitHub makes it very easy to view changes to code. That’s why few people bother with this rule.


The team is strictly following the rules cause it will show in the main repo if merged. Use the github Desktop version, and your commit messages are the same test temp locally in your pc, then push the final version. I don't really want this PR to be rejected.

Same commit message multiple times
![Screenshot_20250726_202613_GitHub.jpg](https://github.com/user-attachments/assets/48b26361-17fc-4ef1-bfab-8ddabf0d895e)

StudioAsInc commented:
> I've compiled all the changes here every time there's a change, right? The latest changes are all improvements over the previous ones. Shouldn't you at least be able to read the code and see what's changed?

Yes, I understand... But they definitely rules 💣 It hurts

AnBui2004 commented:
> > This rule isn’t really useless because GitHub makes it very easy to view changes to code. That’s why few people bother with this rule.
> 
> The team is strictly following the rules cause it will show in the main repo if merged. Use the github Desktop version, and your commit messages are the same test temp locally in your pc, then push the final version. **I don't really want this PR to be rejected.**
> 
> Same commit message multiple times ![Screenshot_20250726_202613_GitHub.jpg](https://private-user-images.githubusercontent.com/177647015/471093553-48b26361-17fc-4ef1-bfab-8ddabf0d895e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM1NDE1MDksIm5iZiI6MTc1MzU0MTIwOSwicGF0aCI6Ii8xNzc2NDcwMTUvNDcxMDkzNTUzLTQ4YjI2MzYxLTE3ZmMtNGVmMS1iZmFiLThkZGFiZjBkODk1ZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNlQxNDQ2NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YTM2NzI5NzY4ZjczYzM1YzMyNDIxNGI4Y2M3ZGE5OWI2YjQwNmVmZjRkY2Q3NmVjOTdmYmJkMjEyYzM3ZjBiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HrSM45RDr2Vca2gK05mgaP-smOZc8jb0YRcGLy8TOts)

Can't read code? Aren't you human? These are small changes based on my previous changes and they don't have any major impact. At least you should be able to read code.

StudioAsInc commented:
> > > This rule isn’t really useless because GitHub makes it very easy to view changes to code. That’s why few people bother with this rule.
> > 
> > The team is strictly following the rules cause it will show in the main repo if merged. Use the github Desktop version, and your commit messages are the same test temp locally in your pc, then push the final version. **I don't really want this PR to be rejected.**
> > 
> > Same commit message multiple times ![Screenshot_20250726_202613_GitHub.jpg](https://private-user-images.githubusercontent.com/177647015/471093553-48b26361-17fc-4ef1-bfab-8ddabf0d895e.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTM1NDE1MDksIm5iZiI6MTc1MzU0MTIwOSwicGF0aCI6Ii8xNzc2NDcwMTUvNDcxMDkzNTUzLTQ4YjI2MzYxLTE3ZmMtNGVmMS1iZmFiLThkZGFiZjBkODk1ZS5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNzI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDcyNlQxNDQ2NDlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03YTM2NzI5NzY4ZjczYzM1YzMyNDIxNGI4Y2M3ZGE5OWI2YjQwNmVmZjRkY2Q3NmVjOTdmYmJkMjEyYzM3ZjBiJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.HrSM45RDr2Vca2gK05mgaP-smOZc8jb0YRcGLy8TOts)
> 
> Can't read code? Aren't you human? These are small changes based on my previous changes, and they don't have any major impact. At least you should be able to read code.

**Bro! I don't have any problems if they merge. it's not even a problem**

AnBui2004 commented:
I suggest changing to another moderator.

AnBui2004 commented:
If it's not approved in 24 hours I lose all changes. Simple as that.

TheRealAshik commented:
@PranavPurwar Why was this rejected, btw?

trindadedev13 commented:
> @PranavPurwar Why was this rejected, btw?

no he just closed the pr

yoann256 commented:
> > @PranavPurwar Why was this rejected, btw?
> 
> no, he just doesn't accept making requested changes

frr

PranavPurwar commented:
> @PranavPurwar Why was this rejected, btw?

I do not know. I didn't reject it.
